lib/constants.dart

```diff
@@ -5,4 +5,36 @@ class Constants {
   static const String settingDisclaimerVersion = 'disclaimer_version';
   static const String settingDisclaimerAgreedDateTime =
       'disclaimer_agreed_date_time';
+
+  /// Analytics screen names avoid changing to keep analytics data constant
+  /// Even if the UI screen name changes try to keep this as analytics name
+  static const String analyticsHomeScreen = 'home';
```

Comment:
@maks This was very much a conscious decision, because one of the poor people who have to review analytics, routes change a lot as the app changes, name changes, re routes, changed routes etc.Which means over a short time the route based method breaks down and it becomes a mess of new, newer, newest, older etc.

So this way the analytics screens should mostly stay fixed and independent as the routes all inevitably change.

Reply:
Hmm, ok, I've never been a consumer of analytics so will defer to those that do :+1: